src/PaymentGateways/ServiceProvider.php

```diff
@@ -34,5 +35,11 @@ public function boot()
         Hooks::addFilter('give_register_gateway', RegisterPaymentGateways::class);
         Hooks::addFilter('give_payment_gateways', RegisterPaymentGatewaySettingsList::class);
         Hooks::addAction('template_redirect', GatewayRoute::class);
+
+        /**
+         * Stripe Checkout Redirect Handler
+         */
+        Hooks::addAction('wp_footer', CheckoutGateway::class, 'maybeHandleRedirect', 99999);
```

Comment:
@kjohnson is there any reason to set a big number for hook priority?

Reply:
@ravinderk I carried over the priority from the legacy implementation. My focus here is getting the payment gateway to use the Gateway API.